code/modules/projectiles/guns/rifles.dm

```diff
@@ -1561,7 +1549,7 @@
 	allowed_ammo_types = list(/obj/item/ammo_magazine/rifle/autosniper)
 	attachable_allowed = list(
 		/obj/item/attachable/autosniperbarrel,
-		/obj/item/attachable/scope/nightvision,
+		/obj/item/attachable/scope/marine,
```

Comment:
as above, like none of these sprite changes have been tested, they're all going to look weird.